src/mono/wasm/wasm.proj

```diff
@@ -137,6 +137,9 @@
       <_EmccLinkFlags Include="-s MODULARIZE=1"/>
       <_EmccLinkFlags Include="-Wl,--allow-undefined"/>
       <_EmccLinkFlags Include="-s ENVIRONMENT=&quot;web,webview,worker,node,shell&quot;" />
+      <!-- remove -Wno-limited-postlink-optimizations once below issue is fixed
+           active issue: https://github.com/emscripten-core/emscripten/pull/16727 -->
```

Comment:
The issue seems to be fixed. And the page says "first appeared in 3.1.11"

Reply:
It is still happening in 3.1.12, for example here https://github.com/dotnet/runtime/runs/6869929890